<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteConfirmDialog</class>
 <widget class="QDialog" name="DeleteConfirmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_askcontent">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm to delete ?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>44</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color:rgba(71,107,253,1);


	border-style:inset;  
	border-radius:3px;  
	color:white;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 10px;
}
QPushButton#pushButton:pressed {  
    /* 改变背景色   
    background-color:rgb(40,85,20); */
    /* 使文字有一点移动 */  
    padding-left:6px;  
    padding-top:6px;  
}  
</string>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>44</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color:#ffffff;
	border:1px solid #D9D9D9;
	border-radius:3px;  
	color:#999999;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 10px;
}
QPushButton#pushButton:pressed {  
    /* 改变背景色   
    background-color:rgb(40,85,20); */
    /* 使文字有一点移动 */  
    padding-left:6px;  
    padding-top:6px;  
}  
</string>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>9</width>
     <height>9</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/modalclose.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_askimg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/ask.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
